host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_nvcc_c++17.cmake

```diff
@@ -59,6 +59,8 @@ set(CMAKE_CUDA_FLAGS "-Xcompiler=--gcc-toolchain=${_GCC_DIR} -restrict -arch ${_
 
 set(CUDA_SEPARABLE_COMPILATION ON CACHE BOOL "" )
 
```

Comment:
```suggestion
```

Reply:
then should we remove `clang@upstream_nvcc_c++17_no_separable.cmake`?

Reply:
Thats a good point.. updated the suggestion to use the real CMake variable.

Reply:
I ended up changing all instances of `CUDA_SEPARABLE_COMPILATION` to `CMAKE_CUDA_SEPARABLE_COMPILATION`

https://github.com/LLNL/blt/pull/629/commits/b0cc1b9497bd1408502f201b16437a9dcf51ae8c